src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,58 @@
+.clock {
+  @include rectangle ($clock-size, $clock-size);
+  background-color: gold;
+  border-radius: 40%;
+
+  position: relative;
+
+  &__round {
+    top: 50%;
+
+    @include rectangle ($round-clock, $round-clock);
+    @extend %watch-hands;
+    background-color: orangered;
+    z-index: 1;
+    transform: translate(-50%, -50%);
+    border-radius: 50%;
+
+  }
+
+  &__seconds {
+    top: $clock-paddings;
+
+    background-color: orangered;
+    border-radius: 6px;
+
+    position: relative;
+
+    @include rectangle (10px, $seconds-hand);
+    @extend %watch-hands;
+    transform: translateX(-50%);
+    transform-origin: bottom right;
+    animation: move 60s linear infinite;
+  }
+
+  &__minutes {
+    top: $clock-paddings + $shortage-mins-to-seconds;
+    background-color: green;
+    border-radius: 6px;
+
+    @include rectangle (15px, $minutes-hand);
+    @extend %watch-hands;
+    transform: translateX(-50%);
+    transform-origin: bottom left;
+    animation: move 3600s linear infinite;
+
+  }
+
+}
+
+@keyframes move {
+  0% {
```

Comment:
You don't have to set deg. for 0%, it works even if you set only 100%)

Reply:
Make sense

Reply:
really works)) "from" was deleted.